### ISSUE TITLE
Crash Apon New World Generation 1.19.4

### ISSUE THREAD

**Issue by Zelioth** [nicguzzo/wands#41]
Minecraft 1.19.4
Forge - 45.0.50
BW - 1.19.4-2.6.6-beta

When having the mod activated the client will crash after the world generated/Joining world. Removing the mod from the pack takes the crash away. 

LOG - https://gist.github.com/Zelioth/b633ca2db55074194317fa309482c967

[crash-2023-05-02_23.42.23-client.txt](https://github.com/nicguzzo/wands/files/11379297/crash-2023-05-02_23.42.23-client.txt)

**Comment by nicguzzo** [comment]
try deleting the config file, make a copy of it before and send it to me, if that works. I've notices that sometimes it gets corrupted and I don't know why

**Comment by Zelioth** [comment]
what would the config be called? I am the dev  and creator of friends United. A somewhat large mod pack. I'm just working on a new 1.19.4 version and it's all a fresh new install. Configs, mods, files all new. I don't see a config for the mod  itself in the config files. it happens on every single world generation and load.

**Comment by Zelioth** [comment]
![Screenshot 2023-05-03 154359](https://user-images.githubusercontent.com/77481431/236045358-2a6af25c-6be3-46be-afc8-84326c5205a2.png)


Could be missing it but I do not see a config in the config files. 

New crash for new generated world and load in 
https://gist.github.com/Zelioth/9ed5412ee4e2732b97950bea70312932

**Comment by Zelioth** [comment]
![Screenshot 2023-05-03 154542](https://user-images.githubusercontent.com/77481431/236045648-fe2c325a-fc30-4fd7-943f-ef8d74524edb.png)

and just to confirm I do have it in my mod pack files.

**Comment by nicguzzo** [comment]
the file should be .minecraft/config/wands.json but I just tested on a new forge 1.19.4 architectury-8.1.79-forge cloth-config-10.0.96-forge and it works fine. It must be some weird interaction with other mod using architectury by the looks of the crash stacktrace, can you identify what other mods depend on architectury? that way I can test them.

**Comment by Zelioth** [comment]
![Screenshot 2023-05-03 182456](https://user-images.githubusercontent.com/77481431/236072402-69e14df0-c115-4f7e-b4a9-dbd0aa0a4682.png)


could it be because I have this mod also? 

https://www.curseforge.com/minecraft/mc-mods/construction-wand

**Comment by nicguzzo** [comment]
no, they can coexists, they have a different modid so no problems.
I'll test with those unchecked mods.
